docs/chapters/workflow/release/release-process.rst

```diff
@@ -11,9 +11,12 @@ licensing reasons, as there are more obligations when doing a binary release.
 
 Binary release
 --------------
-Binary images (starting from Pelux 3.0) are now available for `download`_. Each new release will have a set of binary images supporting the various variants as mentioned below.
+Binary images (starting from Pelux 3.0) are now available for `download`_.
+Each new release will have a set of binary images supporting the various
+variants as mentioned below.
 
-Currently 4 different variants are released. Every hardware offers the qtauto image as well as the minimal image, while QEMU offers only the minimal image. 
+Currently 2 different variants are released: every hardware offers the Qt Auto
```

Comment:
I don't understand why would say only two. I saw @fhajredini's comment, but there is intel, arp, rpi and qemu. That's 4.

Reply:
2 variants: minimal and qt qauto

Reply:
The point of this paragraph was to say we have intel, arp, rpi and qemu (platforms) and for all of those (except) qemu there is the minimal and qtauto variant.

Reply:
So I believe "variant" in the first sentence is a typo.

Reply:
what is the correct word then?

Reply:
Maybe we can say we support 3 hardware (arp, rpi and intel) besides qemu.

Reply:
Platforms.
```suggestion
Currently 4 different platforms are supported. Every hardware offers the qtauto image as well as the minimal image, while QEMU offers only the minimal image.
```
Then the table below shows all the platforms and variants.